src/olympia/users/notifications.py

```diff
@@ -66,9 +66,10 @@ class announcements(_NOTIFICATION):
     id = 8
     group = 'dev'
     short = 'announcements'
-    label = _("add-on contests or events are announced")
+    label = _("stay up-to-date with news and events relevant to add-on "
+              "developers")
```

Comment:
All of the rest of the file used doubles for the text so it was consistent at least.  But I'll change (this one).